HIP/hip-0000-token-update-remove-keys.md

```diff
@@ -0,0 +1,176 @@
+---
+hip: 0000
+title: Remove Existing Keys From A Token
+author: Justyn Spooner <justyn@dovu.io>
+working-group: Justyn Spooner <justyn@dovu.io>
+type: Standards Track
+category: Service
+needs-council-approval: Yes
+status: Draft
+created: 2022-08-05
+discussions-to: https://github.com/hashgraph/hedera-improvement-proposal/discussions/545
+updated: 2022-10-05, 2022-10-12
+requires:
+replaces:
+superseded-by:
```

Comment:
This hip shouldn't need these as it doesn't rely on another hip or replace any existing one.